libs/plugins/rudder-plugin-onetrust-consent-filter-react-native/android/src/main/java/com/rudderstack/plugin/reactnative/onetrust/OnetrustConsentFilterPlugin.kt

```
@@ -0,0 +1,113 @@
+
+    var rudderOneTrustConsentFilter: RudderConsentFilter? = null
+        private set
+
+    fun setupOneTrust() {
+        otPublishersHeadlessSDK.startSDK(
+            "<STORAGE_LOCATION>", //cdn
+            "<DOMAIN_IDENTIFIER>", // domainIdentifier
+            "<LANGUAGE>", // language code
+            OTSdkParams.SdkParamsBuilder.newInstance().build(),
+            object : OTCallback {
+                override fun onSuccess(otSuccessResponse: OTResponse) {}
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)